packages/model/rollback.go

```diff
@@ -26,3 +35,51 @@ func (r *Rollback) Create(transaction *DbTransaction) error {
 func (r *Rollback) Delete() error {
 	return DBConn.Delete(r).Error
 }
+
+// DataMap returns rollback data as map
+func (r *Rollback) DataMap() (map[string]string, error) {
```

Comment:
this method can be interface method for multiple models, but not every model contains field "data". May be this method should be named "ToMap()" or "Map()"?

Reply:
This method returns a map only for a data attribute, so it is called a "DataMap". If we need such an interface, we'll do ".Data().Map()" or ".Data.Map()".